custom_components/immich/hub.py

```diff
@@ -118,7 +118,7 @@ async def list_all_albums(self) -> list[dict]:
         """List all albums."""
         try:
             async with aiohttp.ClientSession() as session:
-                url = urljoin(self.host, "/api/album")
+                url = urljoin(self.host, "/api/album?shared=true")
```

Comment:
Unfortunately `?shared=true` means the API will **only** return the shared albums, so I can't merge this as-is 😕 

Aren't shared albums included in the results by default?

Reply:
no, i have an account who only has shared albums, in that case, the dropdown doesnt even render.